core/web/jobs_controller_test.go

```diff
@@ -182,12 +181,16 @@ func TestJobController_Create_HappyPath(t *testing.T) {
 		},
 		{
 			name: "keeper",
-			toml: testspecs.GenerateKeeperSpec(testspecs.KeeperSpecParams{
-				Name:              "example keeper spec",
-				ContractAddress:   "0x9E40733cC9df84636505f4e6Db28DCa0dC5D1bba",
-				FromAddress:       "0xa8037A20989AFcBC51798de9762b351D63ff462e",
-				ObservationSource: keeper.ExpectedObservationSource,
-			}).Toml(),
+			toml: `
+               type                        = "keeper"
+               schemaVersion               = 3
+               name                        = "example keeper spec"
+               contractAddress             = "0x9E40733cC9df84636505f4e6Db28DCa0dC5D1bba"
+               fromAddress                 = "0xa8037A20989AFcBC51798de9762b351D63ff462e"
+               evmChainId                  = 4
+               minIncomingConfigurations   = 0
+               externalJobID   		 	   =  "123e4567-e89b-12d3-a456-426655440002"
```

Comment:
formatting

Reply:
Sorry, I don't know what changes are wrong with my IDE or what happens, because locally the formatting is fine. And this happened before for other files too. Very weird...